server/interfaces/interfaces.go

```diff
@@ -1564,3 +1564,17 @@ type AtimeUpdater interface {
 	EnqueueByResourceName(ctx context.Context, downloadString string)
 	EnqueueByFindMissingRequest(ctx context.Context, req *repb.FindMissingBlobsRequest)
 }
+
+type CPULeaser interface {
+	// Acquire returns an []int set of CPUs that should be used as a cgroups
+	// cpuset. The returned cancel function *must* be called after a task
+	// has been completed, in order to free these CPUs for other tasks.
+
+	// The CPULeaser will attempt to return CPUs for exclusive use, but this
+	// is not guaranteed. It is the job of the CPULeaser to return the least
+	// loaded CPUs, so that no CPU is overloaded.
+	//
+	// If more CPUs are requested than the total available on the machine,
+	// the returned set of CPUs will be the set available to the machine.
+	Acquire(numCPUs int) (func(), []int)
```

Comment:
nit: maybe flip the order of the return values? I think cancel usually comes second e.g. `ctx, cancel := context.WithCancel(ctx)`

Reply:
Done.